tests/playTests.sh

```diff
@@ -508,6 +508,8 @@ println "- Create dictionary using all samples for both training and testing"
 $ZSTD --train-fastcover=split=100 -r "$TESTDIR"/*.c "$PRGDIR"/*.c
 println "- Create dictionary using f=16"
 $ZSTD --train-fastcover=f=16 -r "$TESTDIR"/*.c "$PRGDIR"/*.c
+println "- Create dictionary using accel=0 (should error)"
+$ZSTD --train-fastcover=accel=0 -r "$TESTDIR"/*.c "$PRGDIR"/*.c
```

Comment:
The comment says it should error, so do you need `&& die` here as well?

Reply:
It looks like it passes with `accel=0`, so either pick a different accel value for the test, or update the comment.